frontend/components/navbar/NavbarLinks.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div
+    :class="cn(navbarLinksVariants({ variant }), props.class)"
+  >
+    <NavbarLink url="#" :variant="firstLinkVariant">
+      Landsat Locator
+    </NavbarLink>
+    <NavbarLink url="#" :variant="linksVariant">
+      Search for data
+    </NavbarLink>
+    <NavbarLink url="#" :variant="linksVariant">
+      About us
+    </NavbarLink>
+    <img
+      v-if="isUserLoggedIn" h="5vh" alt="User profile picture" rounded-full
```

Comment:
Alt is not good idea, if it doesn't load use placeholder, instead of 
![Screenshot from 2024-10-01 20-20-41](https://github.com/user-attachments/assets/79d19339-19a1-405e-b6c9-35fbd2b12997)

Reply:
Alt is also for blind people